01-introduction.Rmd

```diff
@@ -108,18 +112,34 @@ leaflet() %>%
   addAwesomeMarkers(lng = c(-3, 23), lat = c(52, 53), popup = popup)
 ```
 
-It would be difficult to produce Figure \@ref(fig:interactive) with only 4 lines of code in another language, let alone embed the results in an interactive html page (the interactive version can be viewed at [robinlovelace.net/geocompr](http://robinlovelace.net/geocompr/intro.html)), illustrating R's flexibility. 
-
+It would be difficult to produce Figure \@ref(fig:interactive) with only four lines of code in another language, let alone embed the results in an interactive html page (the interactive version can be viewed at [robinlovelace.net/geocompr](http://robinlovelace.net/geocompr/intro.html)), illustrating R's flexibility. 
```

Comment:
Well, I would be careful with the phrasing:
You can do the same using Python (see https://github.com/python-visualization/folium) or Java (https://dukescript.com/javadoc/leaflet4j/).
And of course you could do the same in JavaScript (since leaflet is a JavaScript library) provided you wrote some wrapper functions around the mapping (as did the leaflet and the mapview package in R).
And in fact it is not the power of R but the power of leaflet that lets you quickly create a map. R merely accesses this power through an interface.